frontend/test/metabase/scenarios/admin/settings/settings.cy.spec.js

```diff
@@ -36,15 +36,13 @@ describe("scenarios > admin > settings", () => {
       .type("foo", { delay: 100 })
       .blur();
 
-    cy.wait("@url")
```

Comment:
This double `cy.wait` call was necessary because of the duplicate request, so fixing this issue broke this cy test until I removed it.

Reply:
Great catch!